flecs_ecs_tracing/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "flecs_ecs_tracing"
+version = "0.1.0"
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+flecs_ecs = { version = "0.1.3", path = "../flecs_ecs" }
+fxhash = "0.2.1"
```

Comment:
If #206 lands, then you should change this to be a workspace dependency on `hashbrown` (moving the current hashbrown dependency into the workspace deps) and updating to not use `fxhash`.